README.md

```diff
@@ -79,7 +79,7 @@ Now you can go and play with it as you wish :smile:
 
 👤 **mavhungu rotshidzwa chester**
 - Twitter: [@rotshidzwamavh1](https://twitter.com/rotshidzwamavh1)
-
+- github: [https://github.com/rotshidzw]
 
 ## Procedure - How to use?
 - Install Webpack and webpack-cli
```

Comment:
- [ ] Add in your README how to run tests locally
- [ ] Currently, I am not able to run `tests` please configure well `jest` 
![jest](https://user-images.githubusercontent.com/44745855/178827078-f11695aa-6ede-461e-a3cb-11cf63417696.png)

Reply:
jest not found simply means that you have to install the jest locally
npm install --save-dev jest  or npm install --global jest 
All relevant dependencies need to be installed. This has been run by the previous code review and it succeeded. It was invalidated because my partner needs to make some commits since it is collaborative work even though I drove the project while he was navigating.

 Please find below tests running locally on my machine.
[screenshot.pdf](https://github.com/ogagaoghene/todo-list-structure/files/9105918/screenshot.pdf)

![image](https://user-images.githubusercontent.com/30408871/178830899-bb874c17-ef5b-4565-bd79-47756e6d8e5e.png)